src/screens/DonateScreen/NumberTextField.js

```diff
@@ -35,25 +72,52 @@ class DonateScreen extends React.PureComponent<Props, State> {
   };
 
   render() {
-    const { placeholder, onFocus, ...otherProps } = this.props;
+    const { label, placeholder, onFocus, style, ...otherProps } = this.props;
     const { focused } = this.state;
+
+    // Note on accessibility:
+    // React Native does not support to mark up labels for input fields
+    // properly. See for example: https://github.com/facebook/react-native/issues/14989
+
     return (
-      <TextInput
-        keyboardType="numeric"
-        onBlur={this.onBlur}
-        onFocus={this.onFocus}
-        placeholder={focused ? placeholder : null}
-        placeholderTextColor={mediumGreyColor}
-        style={[styles.base, focused && styles.focused]}
-        underlineColorAndroid={transparent}
-        {...otherProps}
-      />
+      <View
+        accessible
+        accessibilityLabel={getAccessibiltyLabel(
+          label,
+          otherProps.value,
+          focused
+        )}
+        style={style}
+      >
+        <Text
+          type="h4"
+          color="lightNavyBlueColor"
+          importantForAccessibility="no"
```

Comment:
The label is also hidden because it is included in the `accessibilityLabel` of the parent view.